Engine/media/audio/audio.cpp

```diff
@@ -724,6 +791,8 @@ void clear_music_cache() {
 
 }
 
+void play_new_music(int mnum, SOUNDCLIP *music);
```

Comment:
Defect: this is actually defined as static below and will fail to compile on some compilers.